open-bot.yaml

```diff
@@ -85,10 +85,11 @@ rules:
       identifier: "ci-result"
       message: |-
         @{{commit.author.login}} Please review the following output log for errors:
-
+        <details>
```

Comment:
Also add a `summary` tag to specify the head line: `<details><summary>Show errors</summary>`